pkg/cli/cli.go

```diff
@@ -206,24 +206,48 @@ func (c *command) Parent() *command {
 	return &command{p, c.verbose}
 }
 
-func ToPrintableStr(o *RemoteCVD) string {
-	res := fmt.Sprintf("%s (%s)", o.Name, o.Host)
-	res += "\n  " + "Status: " + o.Status
-	adbState := ""
-	if o.ConnStatus != nil {
-		if o.ConnStatus.ADB.Port > 0 {
-			adbState = fmt.Sprintf("127.0.0.1:%d", o.ConnStatus.ADB.Port)
+func WriteListCVDsOutput(w io.Writer, hosts []*RemoteHost) {
+	for _, host := range hosts {
+		fmt.Fprintln(w, hostOutput(host))
+		if len(host.CVDs) == 0 {
+			fmt.Fprintln(w, " ~~Empty~~ ")
+			fmt.Fprintln(w, "")
```

Comment:
nit: It's not technically necessary but a `continue` here would make it clear that an exceptional condition was checked and the rest of the code block won't execute.

Reply:
agree, done.